pkg/controller/release/scheduler_test.go

```diff
@@ -26,6 +30,16 @@ func init() {
 	releaseutil.ConditionsShouldDiscardTimestamps = true
 }
 
+var localFetchChart = func(chartspec *shipper.Chart) (*helmchart.Chart, error) {
+	data, err := ioutil.ReadFile(path.Join("testdata", "simple-0.0.1.tgz"))
```

Comment:
Is this chart name supposed to be hardcoded? I'd expect it to be formed from them chartspec.